src/js/veteran-id-card/containers/VeteranIDCard.jsx

```diff
@@ -34,11 +34,11 @@ class VeteranIDCard extends React.Component {
           window.dataLayer.push({ event: 'vic-authenticated' });
         }
       } else {
-        window.sessionStorage.setItem('vicInitialAuthStatus', 'unauthenticated');
         if (serviceRateLimitedUnauthed) {
           window.dataLayer.push({ event: 'vic-unauthenticated-ratelimited' });
           this.renderEmailCapture = true;
         } else {
+          window.sessionStorage.setItem('vicInitialAuthStatus', 'unauthenticated');
```

Comment:
Did not catch this, but thanks. Should we add a comment above that says something like, set this variable only for users who have made it past the unauthenticated rate-limiting gate and to the sign-in/log-in page, not for users who are sent to the email capture form

Reply:
+1 to adding explanatory comment

Reply:
Awesome, will do! I may change the name of that key in sessionStorage to something more clear, not sure at the moment what that would be though.